packages/calcite-components/src/components/combobox-item/combobox-item.scss

```diff
@@ -1,37 +1,96 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * --calcite-combobox-item-text-color: defines the text color of a combobox item.
+ * --calcite-combobox-item-selection-icon-color: defines the icon color of the dot or custom icon shown at the start of a combobox item.
+ * --calcite-combobox-item-icon-color: defines the color of a custom combobox-item icon
+ * --calcite-combobox-item-box-shadow: defines the box shadow of a combobox item.
```

Comment:
Should box shadow be applied to Combobox Items? or just border?

Reply:
removed.